src/App.js

```diff
@@ -1,10 +1,75 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+import Title from './Title';
+import Candidate from './Candidate';
 
 function App() {
+  const [candidates, setCandidates] = useState([]);
+  const [rerenderTrigger, setRerenderTrigger] = useState(true);
+  let candidatesData = [];
+  let sortedCandidates = [];
+
+  useEffect(() => {
+    const getCandidate = async () => {
+      try {
+        const response = await axios.get(
+          'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates'
+        );
+        candidatesData = response.data;
+      } catch (e) {
+        console.log('Candidates error : ', e);
+      }
+    }
+    
+    async function sortVotes() {
+      sortedCandidates = candidatesData
+        .sort((a, b) => {return b.voteCount - a.voteCount})
```

Comment:
화살표함수에서 바로 return할땐 중괄호와 return문을 생략할 수 있습니다~

```suggestion
       .sort((a, b) => b.voteCount - a.voteCount)
```

Reply:
좀 더 예쁘게 가능했네요 감사합니다 !